modules/dashboard-widgets/module.php

```diff
@@ -0,0 +1,483 @@
+<?php
+
+namespace Elementor\Modules\DashboardWidgets;
+
+use Elementor\Api;
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Plugin;
+use Elementor\TemplateLibrary\Source_Local;
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Class Module
+ *
+ * @package ElementorLabs\Modules\Widgets
+ */
+class Module extends BaseModule {
+
+	/**
+	 * @var \WP_Query $recently_edited_query
+	 */
+	private $recently_edited_query;
+
+	public function get_name() {
+		return 'widgets';
+	}
+
+	public function __construct() {
+		$this->set_vars();
```

Comment:
Why ?

Reply:
I just get this code and improve this. You can see it still in draft.